docker-compose

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# set the docker user to the local user so test environments and artifacts
+# can be access from inside and outside of the test container
+export DOCKER_USER=$(id -u):$(id -g)
```

Comment:
This can be set via `docker-compose.yml`: https://github.com/wemake-services/wemake-django-template/blob/master/%7B%7Bcookiecutter.project_name%7D%7D/docker-compose.override.yml#L23

Reply:
https://github.com/wemake-services/wemake-django-template/blob/master/%7B%7Bcookiecutter.project_name%7D%7D/docker/django/Dockerfile#L13 yes, some extra tweeks are needed.

Reply:
Ah I see! But that only works if the GID and UID of the user that runs the test-suite are `1000` right?

Reply:
I added https://github.com/lona-web-org/lona/pull/414/commits/2b2dbddb13b88bbde461b21746639102f2ab3c92. You are right, it works great. Users that are not `1000:1000` on their system will have to edit `playwright.Dockerfile` and `docker-compose.yml` though. Maybe that's better than a magic script though.